### ISSUE TITLE
googleapiclient import error

### ISSUE THREAD

**Issue by dkafri** [quantumlib/Cirq#2926]
In terminal:
```bash
mkdir test
cd test
python3 -m venv env
source env/bin/activate
pip install cirq
```

I then create the following python script (test_script.py):
```python
import cirq
```

Running the script gives:
```bash
(env) dkafri@dkafri:~/Desktop/test$ python test_script.py 
Traceback (most recent call last):
  File "test_script.py", line 1, in <module>
    import cirq
  File "/usr/local/google/home/dkafri/Desktop/test/env/lib/python3.7/site-packages/cirq/__init__.py", line 38, in <module>
    from cirq import (
  File "/usr/local/google/home/dkafri/Desktop/test/env/lib/python3.7/site-packages/cirq/google/__init__.py", line 42, in <module>
    from cirq.google.engine import (
  File "/usr/local/google/home/dkafri/Desktop/test/env/lib/python3.7/site-packages/cirq/google/engine/__init__.py", line 21, in <module>
    from cirq.google.engine.engine import (
  File "/usr/local/google/home/dkafri/Desktop/test/env/lib/python3.7/site-packages/cirq/google/engine/engine.py", line 38, in <module>
    from apiclient import discovery, http as apiclient_http
  File "/usr/local/google/home/dkafri/Desktop/test/env/lib/python3.7/site-packages/apiclient/__init__.py", line 22, in <module>
    __version__ = googleapiclient.__version__
AttributeError: module 'googleapiclient' has no attribute '__version__'
```

I am able to fix this by downgrading google-api-python-client:
```bash
pip install google-api-python-client==1.7.11
```

**Comment by DuckTigger** [comment]
Seconded - but you only need to downgrade to `google-api-python-client==1.8.0`

**Comment by dkafri** [comment]
> What version of python3 are you using?
3.7.6

**Comment by vtomole** [comment]
Looks like this is due to a new release of https://github.com/googleapis/google-api-python-client/issues/870.

**Comment by dstrain115** [comment]
This issue has already been fixed in the latest version.  You can either downgrade google-api-python-client as mentioned or install cirq-unstable (the latest version).  This will be fixed in the cirq stable version (`pip install cirq`) once we release a new version.

**Comment by dabacon** [comment]
Assume this is now fixed in move to cirq version 0.8.0. Closing, please reopen if I'm wrong
